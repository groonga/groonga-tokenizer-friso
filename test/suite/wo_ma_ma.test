register tokenizers/friso

tokenize TokenFriso "我妈妈"
