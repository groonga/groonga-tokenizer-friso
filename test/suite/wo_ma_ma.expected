register tokenizers/friso
[[0,0.0,0.0],true]
tokenize TokenFriso "我妈妈"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "我",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "妈妈",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
